scripts/uberenv/spack_configs/blueos_3_ppc64le_ib_p9/compilers.yaml

```diff
@@ -51,6 +51,18 @@ compilers:
       fc:  /usr/tce/packages/xl/xl-2019.12.23/bin/xlf2003
     spec: xl@coral
     target: ppc64le
-
+- compiler:
+    environment: {}
+    extra_rpaths: []
+    flags: {}
+    modules: []
+    operating_system: rhel7
+    paths:
+      cc:  /usr/tce/packages/xl/xl-2019.08.20/bin/xlc
```

Comment:
Note, I chose the ``2019.08.20`` compiler drop since that is currently being used by the user.
We can update to newer compiler drops in the near future.